armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -235,25 +239,55 @@ def _isTopDummyBlockPresent(self):
 
     def axiallyExpandAssembly(self):
         """Utilizes assembly linkage to do axial expansion."""
+        # Set up conservation report information (useful for debugging)
+        self.massConservationReport = {}
+        materialMasses = collections.defaultdict(float)
+        componentMasses = collections.defaultdict(float)
+        for b in self.linked.a:
+            for c in getSolidComponents(b):
+                materialMasses[c.material.name] += c.getMass()
+                componentMasses[c.name] += c.getMass()
+        self.massConservationReport[self.linked.a] = (
+            list(materialMasses.keys()) + [None] + list(componentMasses.keys())
+        )
+        self.massConservationReport["pre-exp"] = (
```

Comment:
Might be clearer if you changed this header to `mass pre-exp`?

Reply:
Same with the corresponding `post-exp` entry below.

Reply:
Addressed https://github.com/terrapower/armi/pull/1338/commits/80444f851a16baf40e0a2f54c301c84765f8d7d5